src/tutorial/cli-args.md

```diff
@@ -60,9 +60,9 @@ Furthermore, we can say a bit about their types:
 The pattern is expected to be a string,
 while the second argument is expected to be a path to a file.
 
-In Rust, it is very common to structure programs around the data they deal with
-so this way of looking at CLI arguments fits very well.
-Let's start with this (in file src/main.rs, before fn main() {):
+In Rust, it is common to structure programs around the data they handle, so this
+way of looking at CLI arguments fits very well. Let's start with this (in file
+`src/main.rs`, before `fn main() {`):
```

Comment:
Added backticks but also reformatted the paragraph just a bit so that it renders nicely in the browser (there was an unfortunate word-wrap otherwise).